x/multisig/keeper/keygen.go

```diff
@@ -39,7 +39,7 @@ func (k Keeper) GetKeygenSessionsByExpiry(ctx sdk.Context, expiry int64) []types
 // GetKey returns the key of the given ID
 func (k Keeper) GetKey(ctx sdk.Context, keyID exported.KeyID) (exported.Key, bool) {
 	var key types.Key
-	ok := k.getStore(ctx).Get(keyPrefix.AppendStr(keyID.String()), &key)
+	ok := k.getStore(ctx).Get(keyPrefix.Append(utils.LowerCaseKey(keyID.String())), &key)
```

Comment:
Why lower case key ID? Each key id is treated as unique, and we didn't have a restriction about lower casing before I think